docs/index.rst

```diff
@@ -3,50 +3,204 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-==================
-pAPRika
-==================
 
-pAPRika is a python toolkit for setting up, running, and analyzing free-energy molecular dynamics simulations.
 
-========
+.. role:: green-font
+.. role:: red-font
+.. role:: ignore-width
 
-Index
------
+.. |tick|    replace:: :green-font:`✓`
+.. |cross|   replace:: :red-font:`✕`
+.. |delta|   unicode:: U+0394
+.. |ast|     replace:: :ignore-width:`*`
+.. |dast|    replace:: :ignore-width:`**`
+.. |br| raw:: html
 
+   <br />
+
+.. image :: _static/images/paprika.png
+   :width: 350px
+   :align: left
+
+|
+|
+|
+|
+|
+|
+
+*pAPRika is a python toolkit for setting up, running, and analyzing free-energy molecular dynamics simulations.*
+
+
+===================
+
+
+Theory
+******
+
+
+Binding free-energy
+===================
+
+In the attach-pull-release (APR) method, a guest molecule is physically pulled out of the host molecule in a defined
+path by applying harmonic restraints. The binding free-energy is then computed in terms of the sum of the work required
```

Comment:
I think there's no hyphen here, though I'm not 100% certain.

Reply:
Okay, I'll remove the hyphen.